soc/nxp/imxrt/imxrt7xx/cm33/flash_clock_setup.c

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright 2024 NXP
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <fsl_common.h>
+#include <fsl_clock.h>
+
+#ifdef CONFIG_HAS_MCUX_CACHE
+#include <fsl_cache.h>
+#endif
+
+void BOARD_EnableXspiCache(CACHE64_CTRL_Type *cache)
```

Comment:
```suggestion
void board_enable_xspi_cache(CACHE64_CTRL_Type *cache)
```
Can we follow Zephyr naming style here?

Also, these should be renamed to not include `board`, since they are at the SOC level.